app/controllers/import_controller.rb

```diff
@@ -182,18 +173,18 @@ def validate_headers(format, headers)
     end
   end
 
-  def validate_field(field, value, row_ind)
+  def import_field(field, value, row_ind)
     return value unless VALIDATION[field]
 
     # TODO: Un-comment when required fields are to be checked upon import
     # value = validate_required_field(field, value, row_ind) if VALIDATION[field][:checks].include?(:required)
-    value = validate_enum_field(field, value, row_ind) if VALIDATION[field][:checks].include?(:enum)
-    value = validate_bool_field(field, value, row_ind) if VALIDATION[field][:checks].include?(:bool)
-    value = validate_date_field(field, value, row_ind) if VALIDATION[field][:checks].include?(:date)
-    value = validate_phone_field(field, value, row_ind) if VALIDATION[field][:checks].include?(:phone)
-    value = validate_state_field(field, value, row_ind) if VALIDATION[field][:checks].include?(:state)
-    value = validate_sex_field(field, value, row_ind) if VALIDATION[field][:checks].include?(:sex)
-    value = validate_email_field(field, value, row_ind) if VALIDATION[field][:checks].include?(:email)
+    value = import_enum_field(field, value) if VALIDATION[field][:checks].include?(:enum)
+    value = import_and_validate_bool_field(field, value, row_ind) if VALIDATION[field][:checks].include?(:bool)
+    value = import_date_field(value) if VALIDATION[field][:checks].include?(:date)
+    value = import_phone_field(value) if VALIDATION[field][:checks].include?(:phone)
+    value = import_and_validate_state_field(field, value, row_ind) if VALIDATION[field][:checks].include?(:state)
+    value = import_sex_field(field, value) if VALIDATION[field][:checks].include?(:sex)
+    value = import_email_field(value) if VALIDATION[field][:checks].include?(:email)
     value
```

Comment:
is there a case where replacing this line with `value.blank? ? nil : value` would cause a change in import validation behavior? if not, would we be able to simplify this logic by removing some of these methods that just return `value.blank? ? nil : value`?

Reply:
I couldn't find any way that this would change the behavior, so I've added it in https://github.com/SaraAlert/SaraAlert/pull/582/commits/33d419546a9cb359df9640c01ef170160c5f5284. Now the `import_date_field` and `import_email_field` functions can go, because they don't add anything.

Reply:
Looks like this may have caused some system tests to fail, I will investigate further.

Reply:
Ah I see the problem. The issue with this (as seen by the failing tests) is that some values want to be `''` or `false` rather than `nil`? Moving it out of specific methods like that changes it for all values, and `blank?` returns true on `false`

Reply:
Oh yeah, this is no good for booleans, because if a boolean gets set to `false`, doing `value.blank? ? nil : value`, will then set that boolean back to `nil`, and `false != nil` for several attributes. Are we cool with me just rolling back this change? It is a bit repetitive, but it is at least explicit about how each type of value is handled.

Reply:
ah that makes sense, we can just roll this back for now then

Reply:
Rolled back in https://github.com/SaraAlert/SaraAlert/pull/582/commits/b7aad80d49c36b3ca99713aeccd936cbed9e020d.